sig-etcd/charter.md

```diff
@@ -0,0 +1,63 @@
+# SIG etcd Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Scope
+
+Owns the etcd project and how it is used by Kubernetes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Development of [etcd] and other repositories under [etcd-io organization]
+- Maintenance of [etcd image] packaged with Kubernetes
+
+[etcd]: https://github.com/etcd-io/etcd
+[etcd-io organization]: https://github.com/etcd-io
+[etcd image]: https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd
+
+#### Cross-cutting and Externally Facing Processes
+
+- Specifying, testing and improving the implicit Kubernetes-ETCD Contract, which includes storage requirements, write and delete requirements, read requirements and watch requirements.
+- Release process of etcd and other binaries belonging to [etcd-io organization]
+
+### Out of scope
+
+- Structure of data stored in etcd by Kubernetes components is owned by SIG API Machinery
+
+## Roles and Organization Management
+
+This SIG follows the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Tech Leads
+
+- Release of etcd and other binaries belonging to [etcd-io organization]
+
+### Deviations from [sig-governance]
+
+- SIG etcd's participation in the Kubernetes release cycle is limited by etcd having a different schedule for its releases.
+- SIG etcd communication utilizes pre-existing forums for communication:
```

Comment:
I noticed a google meet link instead of Zoom too?

Reply:
Ah, good catch, we are actually using zoom now. Updated the meeting link.